tests/commands/test_changelog_command.py

```diff
@@ -15,17 +15,6 @@
 from tests.utils import create_file_and_commit, wait_for_tag
 
 
-@pytest.mark.usefixtures("tmp_commitizen_project")
```

Comment:
May i know why is this test removed?

Reply:
This test would be a duplicate of `test_git.py::test_get_log_as_str_list_empty`. 
Also, it seems unnecessary for the changelog command/test to enforce a specific exception since it has no way of handling such an exception.

Reply:
ah got it. Thanks for your explanation!